proto/aggregation.proto

```diff
@@ -43,23 +43,26 @@ message KeyedTransactionMetrics {
 message TransactionAggregationKey {
   bool trace_root = 1;
   
-  string service_version = 2;
+  string service_version = 4;
 
-  string service_runtime_name = 3;
-  string service_runtime_version = 4;
-  string service_language_version = 5;
+  string service_runtime_name = 6;
+  string service_runtime_version = 7;
+  string service_language_version = 8;
 
-  string event_outcome = 6;
+  string event_outcome = 12;
 
-  string transaction_name = 7;
-  string transaction_type = 8;
-  string transaction_result = 9;
+  string transaction_name = 13;
+  string transaction_type = 14;
+  string transaction_result = 15;
 
-  uint32 faas_coldstart = 10;
-  string faas_id = 11;
-  string faas_name = 12;
-  string faas_version = 13;
-  string faas_trigger_type = 14;
+  uint32 faas_coldstart = 16;
+  string faas_id = 17;
+  string faas_name = 18;
+  string faas_version = 19;
+  string faas_trigger_type = 20;
+
+  // 21-29 are reserved for old fields
```

Comment:
Should we use the `reserved` keyword? https://protobuf.dev/programming-guides/proto3/#fieldreserved

Also I suppose we should add `2-3`, `5`, `9-11` in there as well.

Reply:
TIL too! This is nice!